src/main/resources/com/google/api/codegen/java/standalone_sample.snip

```diff
@@ -41,42 +41,278 @@
+  // Do something
+  for ({@apiMethod.listMethod.resourceTypeName} element : future.get().{@apiMethod.listMethod.iterateMethodName}()) {
+    {@processResponse(sample)}
+  }
+@end
+
+@snippet unpagedListCallableMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  while (true) {
+    {@apiMethod.responseTypeName} response = \
+        {@apiMethod.apiVariableName}.{@apiMethod.name}().call(\
+        {@sampleMethodCallArgList(sample.initCode.fieldSettings)});
+    for ({@apiMethod.listMethod.resourceTypeName} element : \
+        {@resourceListGetCall(apiMethod)}) {
+      {@processResponse(sample)}
+    }
+    String nextPageToken = response.getNextPageToken();
+    if (!Strings.isNullOrEmpty(nextPageToken)) {
+      request = request.toBuilder().setPageToken(nextPageToken).build();
+    } else {
+      break;
+    }
+  }
+@end
+
+@snippet resourceListGetCall(apiMethod)
+  @if apiMethod.unpagedListCallableMethod.resourceListParseFunction
+    {@apiMethod.unpagedListCallableMethod.resourceListParseFunction}(\
+      response.{@apiMethod.unpagedListCallableMethod.resourceListGetFunction}())
+  @else
+    response.{@apiMethod.unpagedListCallableMethod.resourceListGetFunction}()
+  @end
+@end
+
+@snippet syncMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  @if apiMethod.hasReturnValue
+    {@apiMethod.responseTypeName} response = {@sampleSyncMethodCall(apiMethod)};
+  @else
+    {@sampleSyncMethodCall(apiMethod)};
+  @end
+@end
+
+@snippet asyncOperationMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  {@apiMethod.responseTypeName} response = {@apiMethod.apiVariableName}.{@apiMethod.name}(\
+      {@sampleMethodCallArgList(sample.initCode.fieldSettings)}).get();
+@end
+
+@snippet callableMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  ApiFuture<{@apiMethod.callableMethod.genericAwareResponseType}> future = {@sampleFutureMethodCall(apiMethod)};
+  // Do something
+  @if apiMethod.hasReturnValue
+    {@apiMethod.responseTypeName} response = future.get();
+  @else
+    future.get();
+  @end
+@end
+
+@snippet asyncOperationCallableMethodSampleCode(apiMethod, sample)
+  {@initCode(sample.initCode)}
+  OperationFuture<{@apiMethod.callableMethod.genericAwareResponseType}> future = {@sampleFutureMethodCall(apiMethod)};
+  // Do something
+  {@apiMethod.responseTypeName} response = future.get();
+@end
+
+@snippet streamingCallableMethodSampleCode(apiMethod, sample)
+  @switch apiMethod.grpcStreamingType
+  @case "BidiStreaming"
+    {@bidiStreamingCallableSampleCode(apiMethod, sample)}
+  @case "ServerStreaming"
+    {@serverStreamingCallableSampleCode(apiMethod, sample)}
+  @case "ClientStreaming"
+    {@clientStreamingCallableSampleCode(apiMethod, sample)}
+  @default
+    $unhandledCase: {@apiMethod.grpcStreamingType}$
+  @end
+@end
+
+@snippet bidiStreamingCallableSampleCode(apiMethod, sample)
```

Comment:
If these are only used in this file, lets make all of these `@private`

In `method_sample.snip`, let's also make the corresponding functions from which you've now branched  `@private` to clarify our intended use cases and  catch errors.